src/Symfony/Component/Cache/Traits/AbstractTrait.php

```diff
@@ -300,4 +300,12 @@ public static function handleUnserializeCallback($class)
     {
         throw new \DomainException('Class not found: '.$class);
     }
+
+    private static function createNewNamespaceVersion(): string
```

Comment:
I'd suggest reverting this, the previous code was fine (unless you spotted an issue?)

Reply:
The issue is a) we are using hex() now and b) we are validating the version key now